spec/lib/embedded_ansible_spec.rb

```diff
@@ -133,6 +165,29 @@
       EvmSpecHelper.create_guid_miq_server_zone
     end
 
+    describe ".api_connection" do
+      around do |example|
+        old_env = ENV["ANSIBLE_SERVICE_NAME"]
```

Comment:
Is there any reason that this ENV variable should exist in the test environment?  I would expect it to look more like:
```ruby
around do |example|
  ENV["ANSIBLE_SERVICE_NAME"] = "ansible-service"
  example.run
  ENV.delete("ANSIBLE_SERVICE_NAME")
end
```